corehq/apps/data_cleaning/forms/cleaning.py

```diff
@@ -212,3 +213,26 @@ def clean(self):
                 return cleaned_data
 
         return cleaned_data
+
+    def create_bulk_edit_change(self):
+        prop_id = self.cleaned_data["clean_prop_id"]
+        action_type = self.cleaned_data["clean_action"]
+        change_kwargs = {
+            EditActionType.REPLACE: {
+                "replace_string": self.cleaned_data["replace_all_string"],
+            },
+            EditActionType.FIND_REPLACE: {
+                "find_string": self.cleaned_data["find_string"],
+                "replace_string": self.cleaned_data["replace_string"],
+                "use_regex": self.cleaned_data["use_regex"],
+            },
+            EditActionType.COPY_REPLACE: {
+                "copy_from_prop_id": self.cleaned_data["copy_from_prop_id"],
+            },
```

Comment:
It looks like `clean_action` already can be one of several things, and probably will grow in the future. The size of the dictionary when we only need one of the blocks feels a bit odd. I don't know if its worth switching to `if/else` statements right now, but [match/case](https://www.reddit.com/r/pythontips/comments/122780u/python_now_has_switch_statements/) might be something to consider when we eventually get on 3.13

Reply:
sure, but for now we can leave it. thanks for the note about match/case in 3.13